expression/builtin_info_vec.go

```diff
@@ -111,11 +111,23 @@ func (b *builtinCurrentRoleSig) vecEvalString(input *chunk.Chunk, result *chunk.
 }
 
 func (b *builtinUserSig) vectorized() bool {
-	return false
+	return true
 }
 
+// evalString evals a builtinUserSig.
+// See https://dev.mysql.com/doc/refman/5.7/en/information-functions.html#function_user
 func (b *builtinUserSig) vecEvalString(input *chunk.Chunk, result *chunk.Column) error {
-	return errors.Errorf("not implemented")
+	n := input.NumRows()
+	data := b.ctx.GetSessionVars()
+	if data == nil || data.User == nil {
```

Comment:
Because our mock context used for testing doesn't have a user-item, it always returns this error.
Please set a user-item for our mock context when creating it at `util/mock/context.go`:
```
	sctx.sessionVars.User = &auth.UserIdentity{
		Username:     "tidb",
		Hostname:     "localhost",
		CurrentUser:  true,
		AuthHostname: "localhost",
		AuthUsername: "tidb",
	}
```

Reply:
got this error

`err1 [planner:1142]INSERT command denied to user 'localhost'@'tidb' for table 't', err2 <nil> for &{for %s [insert into t (a) values (1)]}`

Then I use `root`, it's still error

Reply:
It causes some privilege errors in other tests, so it seems we can't set a user in `NewContext` which is used by other tests.
How about setting this mock user at `benchmarkVectorizedBuiltinFunc` and `testVectorizedBuiltinFunc`, for example: 
```
if funcName == "xxx" {
    ctx.GetSessionVars().User = ...
}
```
Here is an example: https://github.com/pingcap/tidb/blob/master/expression/bench_test.go#L994.